src/script/descriptor.cpp

```diff
@@ -1418,6 +1420,59 @@ class RawTRDescriptor final : public DescriptorImpl
     }
 };
 
+/** A parsed rawnode(...) descriptor */
+class RawNodeDescriptor final : public DescriptorImpl
+{
+    std::vector<unsigned char> m_bytes;
+protected:
+    std::string ToStringExtra() const override { return HexStr(m_bytes); }
+    std::vector<CScript> MakeScripts(const std::vector<CPubKey>&, Span<const CScript>, FlatSigningProvider&) const override { return Vector(CScript(m_bytes.begin(), m_bytes.end())); }
+public:
+    RawNodeDescriptor(std::vector<unsigned char> bytes) : DescriptorImpl({}, "rawnode"), m_bytes(std::move(bytes)) {}
+
+    bool IsSolvable() const final { return false; }
+
+    bool IsSingleType() const final { return true; }
```

Comment:
In "descriptor: Implement rawnode() partial descriptor" a9a9e78c3f839eceebb3abd473d8658fabd92f1f
This can be inlined?